src/main/java/com/gl/donate_receive/donate_receive/controller/HomeController.java

```diff
@@ -0,0 +1,22 @@
+package com.gl.donate_receive.donate_receive.controller;
+
+import com.gl.donate_receive.donate_receive.service.ItemService;
+import org.springframework.stereotype.Controller;
+import org.springframework.ui.Model;
+import org.springframework.web.bind.annotation.GetMapping;
+
+@Controller
+public class HomeController {
+	private final ItemService itemService;
```

Comment:
I would autowire services and repositories. Having a public constructor for a controller or service seems redundant for me.

Reply:
public constructor by default use Autowire annotation. Autowire is not recommended to use throughout the field, because for this, reflection is used, which reduces productivity.